staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/marshal.go

```diff
@@ -59,6 +87,39 @@ func (s *JSONSchemaPropsOrBool) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (s JSONSchemaPropsOrBool) MarshalCBOR() ([]byte, error) {
+	if s.Schema != nil {
+		return cbor.Marshal(s.Schema)
+	}
+	return cbor.Marshal(s.Allows)
+}
+
+func (s *JSONSchemaPropsOrBool) UnmarshalCBOR(data []byte) error {
```

Comment:
make sure all the branches we think are non-error reachable branches are hit in the tests that compare our json / cbor serialized/deserialized objects (applies to all the marshalcbor/unmarshalcbor impls)

Reply:
I checked this just now. All of the reachable non-error cases in "marshal.go" are covered by unstructured roundtrip fuzz testing -- for both JSON and CBOR methods.